common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Availability.scala

```diff
@@ -78,6 +93,27 @@ object Availabilities {
       }
   }
 
+  private implicit class OptionalAccessConditionOps(ac: AccessCondition) {
+    def termsAreOtherInstitution: Boolean =
+      ac.terms match {
+        case Some(t) if t.toLowerCase.contains("available at") => true
+        case Some(t) if t.toLowerCase.contains("available by appointment at") =>
+          true
+
+        case Some(t) if t.contains("Churchill Archives Centre") => true
+        case Some(t) if t.contains("UCL Special Collections and Archives") =>
+          true
+        case Some(t) if t.contains("at King's College London") => true
```

Comment:
Wondering why we have `at` here, but not with the `Churchill Archives Centre`? Not that it's an issue, just made me wonder.

Reply:
Possibly the Churchill Archives string is "in the", or something? I don't remember.